packages/Icon/helper/template.js

```diff
@@ -0,0 +1,12 @@
+module.exports = ({ template }, opts, { imports, interfaces, componentName, jsx, exports }) => {
```

Comment:
maybe would worth putting a link or reference where to read more about it? so the next developer can get an idea of how this works?